test/e2e/autotls/config/util.go

```diff
@@ -28,12 +28,12 @@ import (
 )
 
 type EnvConfig struct {
-	FullHostName                  string `envconfig:"full_host_name" required:"true"`
-	DomainName                    string `envconfig:"domain_name" required:"true"`
-	DNSZone                       string `envconfig:"dns_zone" required:"true"`
-	CloudDNSServiceAccountKeyFile string `envconfig:"cloud_dns_service_account_key_file" required:"true"`
-	CloudDNSProject               string `envconfig:"cloud_dns_project" required:"true"`
-	IngressIP                     string `envconfig:"ingress_ip" required:"true"`
+	AutoTLSTestFullHostName                  string `envconfig:"auto_tls_test_full_host_name" required:"true"`
```

Comment:
I _think_ you can use `envconfig.Process("auto_tls_test", ..)` in main.go and then you don't need to prefix these all individually here

Reply:
Yeah, it allows prefix for the vars.

Reply:
Good point. Done